src/nft/css/sprinkles.css.ts

```diff
@@ -41,6 +41,8 @@ const themeContractValues = {
   opacity: {
     hover: '0.6',
     pressed: '0.4',
+    disabled: '0.5',
+    enabled: '1',
```

Comment:
Do you need to add these to the sprinkles theme? Looks like you use the shared styles from `styled.d.ts` in the component

Reply:
ur right i don't need this, removed. btw also added in the tooltip fixes